Base/QTCore/qSlicerSlicer2SceneReader.cxx

```diff
@@ -1219,8 +1219,8 @@ void qSlicerSlicer2SceneReaderPrivate::importOptionsNode(NodeType& node)
   Q_Q(qSlicerSlicer2SceneReader);
   //$::slicer3::MRMLScene SetErrorMessage "warning: option nodes cannot be imported"
   //$::slicer3::MRMLScene SetErrorCode 1
-  q->mrmlScene()->SetErrorMessage("Warning: option nodes cannot be imported");
-  q->mrmlScene()->SetErrorCode(1);
+  //q->mrmlScene()->SetErrorMessage("Warning: option nodes cannot be imported");
+  //q->mrmlScene()->SetErrorCode(1);
```

Comment:
At some point we should deprecate slicer2 scenes and remove this dead code.

Reply:
I've removed it now. It was not tested and was surely broken. Slicer2 scenes can be better converted to Slicer3 scenes using Slicer3 application.